File: JAVA/Pertemuan 01/src/id/ac/uph/pbo/pertemuan01/Buku.java
package id.ac.uph.pbo.pertemuan01;

public class Buku {

    private String judul;
    private String penulis;
    private int    jumlahHalaman;

    public Buku(String judul, String penulis, int jumlahHalaman) {
        this.judul         = judul;
        this.penulis       = penulis;
        this.jumlahHalaman = jumlahHalaman;
    }

    // Method yang diminta slide
    public void tampilkanInfo() {
        System.out.println("  ┌─────────────────────────────────────────┐");
        System.out.println("  │ Judul   : " + judul);
        System.out.println("  │ Penulis : " + penulis);
        System.out.println("  │ Halaman : " + jumlahHalaman);
        System.out.println("  └─────────────────────────────────────────┘");
    }
}

File: PYTHON/Pertemuan 01/Buku.py
import sys
sys.stdout.reconfigure(encoding='utf-8')
class Buku:
    def __init__(self, judul: str, penulis: str, jumlah_halaman: int):
        self.judul = judul
        self.penulis = penulis
        self.jumlah_halaman = jumlah_halaman

    # Method yang diminta slide
    def tampilkan_info(self):
        print("  ┌─────────────────────────────────────────┐")
        print(f"  │ Judul   : {self.judul}")
        print(f"  │ Penulis : {self.penulis}")
        print(f"  │ Halaman : {self.jumlah_halaman}")
        print("  └─────────────────────────────────────────┘")

